r3bsource/base/R3BUcesbSource.cxx

```diff
@@ -311,19 +310,19 @@ void R3BUcesbSource::Close()
     if (0 != ret)
     {
         perror("ext_data_clnt::close()");
-        R3BLOG(fatal, "ext_data_clnt::close() failed");
+//        R3BLOG(fatal, "ext_data_clnt::close() failed");
     }
 
     /* Close pipe */
-    if (nullptr != fFd)
+    if (0 && nullptr != fFd)
     {
         int status;
         status = pclose(fFd);
         if (-1 == status)
         {
             perror("pclose()");
             R3BLOG(fatal, "pclose() failed");
-            abort();
+            //abort();
```

Comment:
why?

Reply:
R3BUcesbSource is a general class used for unpacking the data, so please go to the right version.